content/news/2021-08-19-gsoc-tools.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Devin's Google Summer of Code Ending"
+date = 2021-08-19
+extra.author = "Devin Ragotzy"
++++
+
+# A Summer of Ruma Tooling Comes to an End
+
+What a fun summer! I was hoping, this year I would become more a part of the Rust community, and
+I think I have. I learned how to open an MCP (Major Change Proposal), which is how Rust makes
+smaller changes to the language. I am involved in an effort to create a new plugin lint system
+similar to [Clippy]. I made a few more contributions to Clippy, fixing old lints and adding a new one.
+I am happy to become more involved in Ruma and Rust and plan to finish all my works in progress.
+
+Since I am working with more people than just my mentor, making progress seems to take longer.
+I am glad for the experience working with multiple people, learning how to follow directions from
+multiple people. You have to be able to write code in the "voice" of someone else or at least
+meet them halfway. I know that this is an invaluable skill that will serve me well in my career.
+Thank you to [@jplatte] and [@iinuwa] for all the help and encouragement!
+
+## Starting on Automation
+
+- [cargo-sort][DevinR528/cargo-sort] Some of the work on this happened before GSoC started but not all
+(I'm proud of this crate)
+- [nonstandard-macro-braces][rust-lang/rust-clippy#7299] Lint for consistent macro brace usage
+- [import-renaming][rust-lang/rust-clippy#7300] Lint to enforce import renaming
+- [disallow-types][rust-lang/rust-clippy#7315] Lint to ban specified types
+- [fix FP][rust-lang/rust-clippy#7431] Fix false positive in macro brace lint
+
+## Work During the Second Half
+
+- [module layout style][rust-lang/rust-clippy#7543] Clippy now has the ability to lint on the module
+layout chosen
+- [non_exhaustive reachable lint][rust-lang/rust#86809] I've been busy continuing work on this pull
+request
+- [plugin lint design][lint/design] I spent a few days working with a group of people designing a new 
+inter tool that could load third-party lints
```

Comment:
Missing `l` at the start, also I think `linter tool` is kind of a weird intersection between `linter` and `linting tool` and you might be better off using one of those terms.